go-controller/pkg/util/multi_network.go

```diff
@@ -943,7 +943,7 @@ func GetPodNADToNetworkMappingWithActiveNetwork(pod *kapi.Pod, nInfo NetInfo, ac
 	// Add the active network to the NSE map if it is configured
 	activeNetworkNADs := activeNetwork.GetNADs()
 	if len(activeNetworkNADs) < 1 {
-		return false, nil, fmt.Errorf("missing NADs at active network '%s' for namesapce '%s'", activeNetwork.GetNetworkName(), pod.Namespace)
+		return false, nil, fmt.Errorf("missing NADs at active network '%s' for namespace '%s'", activeNetwork.GetNetworkName(), pod.Namespace)
```

Comment:
nit: you can just use `%q` instead of `'%s'`

Reply:
i just fixed the spelling error in namespace, but i'll also fix the quotes